bin/snare

```diff
@@ -176,26 +234,38 @@ if __name__ == '__main__':
 
     with open(os.path.join(full_page_path, 'meta.json')) as meta:
         meta_info = json.load(meta)
+<<<<<<< HEAD
```

Comment:
And this

Reply:
@afeena thanks for pointing out made the changes.